src/rules/selector-max-specificity/README.md

```diff
@@ -0,0 +1,29 @@
+# selector-max-specificity
+
+Control the specificity of selectors.
+
+First define the maximum allowable selector specificity as computed by the [W3C specificity calculator](https://www.w3.org/TR/selectors/#specificity):
```

Comment:
That link describes the rules of the calculation but isn't itself a "calculator" (is there one? that would be handy ...)